### PR TITLE
Humairaa chi zm update 0421

### PR DESCRIPTION
**Pull request by Humairaa127**

### CODE REVIEW

**Comment by nickhurlburt** [comment]
@GPaoloni -- does this need a new version?

**Review by GPaoloni — APPROVED** [review]
Looks good! No new version needed for the given changes.

File: plugin-hrm-form/src/formDefinitions/v1/tabbedForms/ChildInformationTab.json [hunk 1]
```diff
@@ -334,6 +337,11 @@
     "name": "LGBTQI+",
     "label": "LGBTQI+ / SOGIESC child",
     "type": "mixed-checkbox"
+  },
+    {
+    "name": "Other Vulnerable group",
```
Comment (nickhurlburt) [inline plugin-hrm-form/src/formDefinitions/v1/tabbedForms/ChildInformationTab.json:342]: For the `name` we've been using `camelCase`, so this should be `otherVulnerableGroup`

File: plugin-hrm-form/src/formDefinitions/v1/tabbedForms/ChildInformationTab.json [hunk 2]
```diff
@@ -334,6 +337,11 @@
...
+    "label": "Other Vulnerable group",
```
Comment (nickhurlburt) [inline plugin-hrm-form/src/formDefinitions/v1/tabbedForms/ChildInformationTab.json:343]: Just to double check, is this the capitalization we want?  Do we want to display it alternatively as "Other vulnerable group" or "Other Vulnerable Group"?

File: plugin-hrm-form/src/formDefinitions/v1/tabbedForms/IssueCategorizationTab.json [hunk 3]
```diff
@@ -112,7 +112,7 @@
   "Sexuality": {
     "color": "#506BA5",
     "subcategories": [
-      "Sexuality and gender identity",
+      "Sexuality orientation and gender identity",
```
Comment (nickhurlburt) [inline plugin-hrm-form/src/formDefinitions/v1/tabbedForms/IssueCategorizationTab.json:115]: Is this what you want here?  "Sexuality orientation and gender identity"?  That doesn't seem like correct grammar.  "Sexuality, orientation and gender identity" or "Sexual orientation and gender identity" would make more sense.

File: plugin-hrm-form/src/formDefinitions/v1/tabbedForms/ChildInformationTab.json (hunk 1 above)
Comment (dee-luo) [reply plugin-hrm-form/src/formDefinitions/v1/tabbedForms/ChildInformationTab.json:342]: @Humairaa127 we'll take this out based on Joan's feedback

File: plugin-hrm-form/src/formDefinitions/v1/tabbedForms/CallerInformationTab.json [hunk 4]
```diff
@@ -17,11 +17,15 @@
...
     "options": [
       { "value": "Unknown", "label": "" },
+      { "value": "Caregiver", "label": "Caregiver" },
       { "value": "Friend", "label": "Friend" },
       { "value": "Grandparent", "label": "Grandparent" },
       { "value": "Neighbour", "label": "Neighbour" },
       { "value": "Parent", "label": "Parent" },
+      { "value": "Partner", "label": "Partner" },
       { "value": "Person in a position of responsibility", "label": "Person in a position of responsibility" },
+      { "value": "Sibling", "label": "Sibling" },
```
[the remark this answers is not in the document]
Comment (dee-luo) [reply plugin-hrm-form/src/formDefinitions/v1/tabbedForms/CallerInformationTab.json:27]: @Humairaa127 small edit here

File: plugin-hrm-form/src/formDefinitions/v1/tabbedForms/IssueCategorizationTab.json (hunk 3 above)
Comment (dee-luo) [reply plugin-hrm-form/src/formDefinitions/v1/tabbedForms/IssueCategorizationTab.json:115]: @Humairaa127 small edit here as well

**Pull request merged**